db.js

```diff
@@ -37,10 +37,32 @@ var client = couchdb.createClient(
     settings.username,
     settings.password
 );
-module.exports = client.db(settings.db);
+var db = client.db(settings.db);
+
+// Fix for 0.4 : https://github.com/medic/medic-projects/issues/1178#issuecomment-273550990
+var nativeViewFunc = db.view;
+db.view = function(design, view, query, cb) {
+    return nativeViewFunc.call(db, design, view, query, function(err, data) {
+        if (!err && !data) {
+            return cb(new Error('Both err and data are undefined'));
```

Comment:
Do the calling functions still throw when they get this?  Just want to make sure we're still not crashing when this is called.